workspace/data/spam.txt

```diff
@@ -0,0 +1,5 @@
+Dear sir, I am a Prince in a far kingdom you have not heard of.  I want to send you money via wire transfer so please ...
+Get Viagra real cheap!  Send money right away to ...
+Oh my gosh you can be really strong too with these drugs found in the rainforest. Get them cheap right now ...
+YOUR COMPUTER HAS BEEN INFECTED!  YOU MUST RESET YOUR PASSWORD.  Reply to this email with your password and SSN ...
+THIS IS NOT A SCAM!  Send money and get access to awesome stuff really cheap and never have to ...
```

Comment:
I doubt we need the `workspace/` directory committed in the repo. Especially this file

Reply:
Done

Reply:
You didn't push anything.

Reply:
Pushed